<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetIpDialog</class>
 <widget class="QDialog" name="SetIpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>42</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头ip：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>102</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>162</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>用户姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>222</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>用户密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_ip">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_port">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_user">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_password">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="set_yes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_no">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_default">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
